definition/voip/VoIpCallerInfo.ts

```diff
@@ -7,18 +7,8 @@ export interface IState {
 	enableVideo: boolean;
 }
 
-export type VoIpCallerInfo =
-	| {
-			state: Exclude<CallStates, 'IN_CALL' | 'OFFER_RECEIVED'>;
-			userState: UserState;
-	  }
-	| {
-			state: 'OFFER_RECEIVED';
-			userState: UserState;
-			callerInfo: ICallerInfo;
-	  }
-	| {
-			state: 'IN_CALL';
-			userState: UserState;
-			callerInfo: ICallerInfo;
-	  }; // TODO: Check for additional properties and States (E.g. call on hold, muted, etc)
```

Comment:
@tiagoevanp I am not sure why these changes were done in this way. These changes were added https://github.com/RocketChat/Rocket.Chat/pull/23801/files#diff-d9b296df208a294ef147f00110a129467d20958e462293be58b54d333fb6f02a
So @MartinSchoeler may be able to put some light on this.